### PR TITLE
__debconf_set_selections: add --reconfigure and --onchange

### PR DESCRIPTION
**Pull request by 4nd3r**

### CODE REVIEW

File: type/__debconf_set_selections/gencode-remote [hunk 1]
```diff
@@ -51,4 +51,15 @@ then
 		{
 			printf "set %s %s %s %s\n", $1, $2, $3, $4
 		}' "${filename}" >>"${__messages_out:?}"
+
+	if test -f "${__object:?}/parameter/reconfigure"
+	then
+		package="$(awk '{print $1}' "${filename}")"
```
Comment (sideeffect42) [inline type/__debconf_set_selections/gencode-remote:57]: 🤔 what if a user updates debconf entries for more than one package at once?

Is there a way to figure out which packages would need to be reconfigured?

Comment (4nd3r) [reply type/__debconf_set_selections/gencode-remote:57]: Then they will use `require=""` and `__check_messages`?

Comment (4nd3r) [reply type/__debconf_set_selections/gencode-remote:57]: Or we can loop over...

Comment (4nd3r) [reply type/__debconf_set_selections/gencode-remote]: A while loop can help here.

**Review by sideeffect42 — APPROVED** [review]

File: type/__debconf_set_selections/gencode-remote [hunk 2]
```diff
@@ -51,4 +51,17 @@ then
 		{
 			printf "set %s %s %s %s\n", $1, $2, $3, $4
 		}' "${filename}" >>"${__messages_out:?}"
+
+	if test -f "${__object:?}/parameter/reconfigure"
+	then
+		awk '{print $1}' "${filename}" | sort | uniq | while read -r package
```
[the remark this answers is not in the document]
Comment (4nd3r) [reply type/__debconf_set_selections/gencode-remote]: b-b-but you already... okay :man_dancing:

**Pull request merged**